Why does redux-devtools need to be installed locally?

Hi Gary,

Just a question please: I saw the instruction about installing `redux-devtools` locally, but I figured I could just point to the `next` branch by configuring NPM like so:

    "redux-devtools": "https://github.com/gaearon/redux-devtools.git#next"

`npm install` ran fine and it appeared to be there under `node_modules`. But running it up with `npm run dev` gave:

```
ERROR in ./src/r3test.js
Module not found: Error: Cannot resolve module 'redux-devtools' in /Users/peter/dev/src/misc/forks/r3test-fork/src
 @ ./src/r3test.js 40:21-46

ERROR in ./~/redux-devtools-log-monitor/lib/LogMonitor.js
Module not found: Error: Cannot resolve module 'redux-devtools' in /Users/peter/dev/src/misc/forks/r3test-fork/node_modules/redux-devtools-log-monitor/lib
 @ ./~/redux-devtools-log-monitor/lib/LogMonitor.js 37:21-46
```

Then I tried a [local npm pack](http://podefr.tumblr.com/post/30488475488/locally-test-your-npm-modules-without-publishing) and that seems to be working fine. I haven't been able to work out why yet though - please might you be able to explain?

Thanks!

--- a/package.json
+++ b/package.json
@@ -20,8 +20,8 @@
   "main": "r3test.js",
   "dependencies": {
     "fbjs": "^0.2.1",
-    "react": "0.14.0",
-    "react-dom": "0.14.0",
+    "react": "0.14.2",
+    "react-dom": "0.14.2",
     "react-redux": "^4.0.0",
     "react-three": "^0.7.2",
     "redux": "^3.0.2",
@@ -36,6 +36,7 @@
     "eslint-plugin-react": "^3.6.3",
     "lodash": "^3.3.1",
     "react-hot-loader": "^1.3.0",
+    "redux-devtools": "^3.0.0-beta-3",
     "redux-devtools-dock-monitor": "^1.0.0-beta-3",
     "redux-devtools-log-monitor": "^1.0.0-beta-3",
     "redux-thunk": "^1.0.0",

